utils/spider.py

```diff
@@ -482,11 +459,6 @@ def _try_spider( time_dict, dirs, COUPLER_options, loop_counter, runtime_helpfil
             else:
                 UpdateStatusfile(dirs, 20)
                 raise Exception("Invalid time-stepping method '%d'" % COUPLER_options["dt_method"])
-            
-            # Additional step-size ceiling when F_crit is used
-            if abs(run_atm.iloc[-1]["F_atm"]) <= COUPLER_options["F_crit"]:
```

Comment:
I think it would be better to keep this in, even though not actively using it. Having step size control as a function of both atmospheric flux AND melt fraction change seems good to me.

Reply:
I agree that this might be a useful thing to include. However, would it be possible to add this back to the code at a later time? I don't want to modify the code before merging this PR, since it's the version that grid was run with (for the paper).